decentraland/unity-renderer
# feat: Start Menu improvements

## What does this PR change?
Fix #1755 

- [x] Player name shouldn't show the address tag when it is a claimed name.
- [x] Clicking on the black side of the header while in Map, triggers a Click that does nothing.
- [x] World must be renderer somehow while changing graphic settings (maybe a little screen or window) -> **_DEACTIVATED TEMPORALLY_**
- [x] When we press ESC when a card modal is open (for example: place or event cards), should be closed only that modal and not the whole Start Menu.
- [x] When pressing ESC in-world without anything else opened, it should open the start menu.
- [x] Remove the StartMenu tutorial step for users that already did the tutorial.

## How to test the changes?
1. Go to: https://play.decentraland.zone/?renderer-branch=feat/Start-Menu-list-of-improvements
2. Test and check each point described above.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
